kind-config.yaml

```diff
@@ -0,0 +1,10 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+  extraPortMappings:
+  - containerPort: 30081
+    hostPort: 8081
+  - containerPort: 30443
+    hostPort: 443
```

Comment:
nit: Since this is Kubernetes-specific, maybe move it to deploy/kubernetes/kind?